simp_le.py

```diff
@@ -1183,5 +1210,33 @@ def test_error_exit_codes(self, dummy_stderr):
                 assert False
 
 
+class IOPluginTests(TestCase):
```

Comment:
This was actually a deliberate choice, the user interface is "key.pem", so if for some reason this was changed, it should still work (as the user expects). I'll change it anyway while making it specific for KeyFilePlugin (originally I intended to check account_key too).